firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -74,24 +74,30 @@ class Setting: NSObject {
         cell.detailTextLabel?.assign(attributed: status, theme: theme)
         cell.detailTextLabel?.attributedText = status
         cell.detailTextLabel?.numberOfLines = 0
-        cell.textLabel?.assign(attributed: title, theme: theme)
-        cell.textLabel?.textAlignment = textAlignment
-        cell.textLabel?.numberOfLines = 0
-        cell.textLabel?.lineBreakMode = .byTruncatingTail
+        if let cell = cell as? ThemedCenteredTableViewCell {
+            cell.applyTheme(theme: theme)
+            cell.setTitle(to: .SettingsDisconnectSyncButton)
+            cell.accessibilityLabel = .SettingsDisconnectSyncButton
```

Comment:
This is a class that is used for a lot of settings so setting a specific label for disconnecting sync should not be done here. Instead call `super.init(title: NSAttributedString(string: .SettingsDisconnectSyncButton))` in `init(settings:)` of `DisconnectSetting` and change the code here like this:

```suggestion
            if let title = title?.string {
                cell.setTitle(to: title)
                cell.accessibilityLabel = title
            }
```